### DISCUSSION TITLE
Found this odd.

### DISCUSSION THREAD

**Discussion by Androxilogin** [sanni/cartreader#326] [General Chat]
I saw **[this video](https://www.youtube.com/watch?v=k_aNSev7Y0M)** pop up in my YouTube feed and wondering if the creator actually spoke with Sanni or _someone else_? Seemed strange to me since he mentioned https://twitter.com/cartridgereader which is all in Japanese. 

EDIT:

I just found that this has been addressed in the forum but for traffic coming from the video itself, a little reference point:

https://forum.arduino.cc/t/rom-reader-for-super-nintendo-super-famicom-game-cartridges/155260/1481

**Comment by SAVE-THE-HERO** [comment]
This is obviously our mistake.
No matter what we say, it will be an excuse.
I asked them to put a correction at the beginning of the sentence in the video description: [Every Retro Game Collector Needs This! The All-In-One Device I've Been Waiting For! - YouTube](https://www.youtube.com/watch?v=k_aNSev7Y0M)
I will continue to give People the correct information too.
https://twitter.com/CartridgeReader/status/1489555753640013830?s=20&t=UDY1GST7ljwl1lzeqTV6qg

I had two requests for him to make the video.
"I have two requests about the video you are making.
Cartridge Reader is an open source project on Github 2.
2. cartridge reader design by sanni & great contributors"
That was conveyed incorrectly.
<img width="452" alt="スクリーンショット 2022-02-04 21 47 08" src="https://user-images.githubusercontent.com/89830526/152531975-5ca6e0dc-c7ae-41b6-b28f-bca1b238f074.png">

No matter what I say, it is an excuse.
I am so sorry.
I pledge to continue to pass on the correct information.
